### PR TITLE
Updated version to 4.3.18 according to recent changes and added replace in files for unix in docugen.

### PR DESCRIPTION
**Pull request by mastep84**
…ed volume selection).

### CODE REVIEW

File: recent_changes.txt [hunk 1]
```diff
@@ -8,6 +8,13 @@ Recent changes:
 
 
 
+-------------------------------------------------------------------------------
+v 4.3.19
+Changes and additions:
+- added Selection-SelectVolumesByType
```
Comment (sreiter) [inline recent_changes.txt]: Please add your changes to the "Changes and additions" of v 4.3.18, which is the next version which will be released. Please also remove the v 4.3.19 section for now - we will add it later when 4.3.18 is done.
Thanks!

File: version.txt [hunk 2]
```diff
@@ -1 +1 @@
-4.3.17
\ No newline at end of file
+4.3.19
```
Comment (sreiter) [inline version.txt]: Please increase to 4.3.18 instead of 4.3.17.
Thanks!

**Review by sreiter — CHANGES_REQUESTED** [review]
4.3.18 has not yet been released - so this would be the next version. Your changes would ideally go into that version.
Thanks

**Comment by mastep84** [comment]
Thank you, applied changes according to your requests.

**Comment by mastep84** [comment]
Now also adding ReplaceInFile functionality for UNIX.

**Review by sreiter — COMMENTED** [review]
Great, thanks!
If it would have been even more comfortable to merge it, if you had created a new the pull request for the docugen related stuff.
Nevertheless, thanks a lot!

**Pull request merged**